pkg/api/dashboard.go

```diff
@@ -1240,3 +1375,14 @@ type DashboardVersionResponse struct {
 	// in: body
 	Body *dashver.DashboardVersionMeta `json:"body"`
 }
+
+// swagger:parameters restoreDeletedDashboardByUID
+type RestoreDeletedDashboardByUID struct {
+	// in:path
+	// required:true
+	UID string `json:"uid"`
+
+	// in:body
+	// required:true
+	Body dashboards.SaveDashboardCommand
```

Comment:
the handler expects `dashboards.RestoreDeletedDashboardCommand` instead:
```suggestion
	Body dashboards.RestoreDeletedDashboardCommand
```